packages/governance/src/types.js

```diff
@@ -445,6 +463,16 @@
  * @property {CreateQuestion} createQuestion
  */
 
+/**
+ * @typedef {object} GovernorFacet
```

Comment:
I was surprised we didn't have this type yet. Reading through https://github.com/Agoric/agoric-sdk/blob/a866f983a6a14c8b87ecc933658773e7afe350c5/packages/governance/README.md?plain=1#L235-L240 I think we'd benefit from defining an interface.

We can't have a formal interface with `.ts` syntax but we can define it as a typedef with a comment to explain.

E.g.,
```js
/**
 * @typedef GovernableCreatorFacet
 * @property {() => ParamManagerRetriever} getParamMgrRetriever
 * @property {(name: string) => Promise<Invitation>} getInvitation
 * @property {() => unknown} getLimitedCreatorFacet
 */
```

Then this would extend that,
```suggestion
 * @typedef {GovernableCreatorFacet} GovernorFacet
```

Reply:
GovernedCreatorFacet is the def'n you're looking for.

I added CF as a parameterized type to GovernorFacet.